app/controllers/application_controller.rb

```diff
@@ -158,6 +159,14 @@ def set_cache_headers
     response.headers["Pragma"] = "no-cache"
     response.headers["Expires"] = "Fri, 01 Jan 1990 00:00:00 GMT"
   end
+
+  def check_disabled_user
+    return unless current_spree_user.disabled
+
+    flash[:success] = nil
```

Comment:
I need to put this one to `nil` as we may have an other message especially while logging in.